php/cross_service/aurora_item_tracker/public/index.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Illuminate\Contracts\Http\Kernel;
+use Illuminate\Http\Request;
+
+define('LARAVEL_START', microtime(true));
+
+/*
+|--------------------------------------------------------------------------
+| Check If The Application Is Under Maintenance
+|--------------------------------------------------------------------------
+|
+| If the application is in maintenance / demo mode via the "down" command
+| we will load this file so that any pre-rendered content can be shown
+| instead of starting the framework, which could cause an exception.
```

Comment:
If the application is in maintenance / demo mode because of the "down" command,
we load this file to show any pre-rendered content instead of starting the framework. 
Starting the framework could cause an exception.

Reply:
external file